experiences/package.json

```diff
@@ -20,8 +20,8 @@
     "build:update-revision": "git rev-parse HEAD > static/git-revision.txt",
     "build:static": "nuxt generate",
     "build:server": "nuxt build",
-    "lint": "eslint --ext .js,.vue",
-    "lint:fix": "eslint --ext .js,.vue --fix",
+    "lint": "npx eslint . --ext .js,.vue",
+    "lint:fix": "npx eslint . --ext .js,.vue --fix",
```

Comment:
Why this change?

Reply:
The previous command did not work on Andreas laptop